.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 version: 2.1
 
 orbs:
-  hmpps: ministryofjustice/hmpps@7
```

Comment:
Hoping this will fix the failing [hmpps/npm_security_audit](https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-activities-management/5904/workflows/4b37c9aa-ecfe-40c1-9ed4-602ac5222234/jobs/24509?invite=true#step-103-0_40) job. If it causes problems - I will revert.